pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -240,6 +282,18 @@ func (cm *containerManagerImpl) setupNode() error {
 		return err
 	}
 
+	// Setup top level qos containers only if EnableQosCgroups flag is specified as true
+	if cm.NodeConfig.EnableQosCgroups {
+		qosConfig := &QOSConfig{
+			RootContainerName: cm.NodeConfig.CgroupRoot,
+		}
+		qosContainersInfo, err := InitQOS(qosConfig)
```

Comment:
If you are passing only one parameter why create a config object to pass that? Why not pass `RootContainerName` as an argument directly?